CONTRIBUTING.md

```diff
@@ -10,13 +10,9 @@ By contributing you will agree that your contribution will be put under the same
 - Contributions
 - Quality
 
-## CLA
-
-Before you can contribute, you will need to sign our cla [Contributor License Agreement](https://github.com/Daimler/daimler-foss/blob/master/cla/2019-09-11_Daimler_FOSS_CLA_DaimlerTSS.pdf) and send the signed CLA to <cla-tss@daimler.com> 
-
```

Comment:
## CLA
Please sign our [Contributor's License Agreement](https://github.com/mercedes-benz/foss/blob/master/CONTRIBUTORS_LICENSE_AGREEMENT.md) otherwise we cannot accept your contribution.

Reply:
Please add the CLA part I wrote before. It is similar to the SecHub CLA part